pkg/cli/alpha/config-gen/cmd.go

```diff
@@ -148,54 +148,68 @@ The KubebuilderConfigGen resource has the following fields:
     crds:
       # path to go module source directory provided to controller-gen libraries
       # optional -- defaults to '.'
-	  sourceDirectory: ./relative/path
+      sourceDirectory: ./relative/path
 
-	# configure how the controller-manager is generated
+    # configure how the controller-manager is generated
     controllerManager:
-	  # image to run
-	  image: my-org/my-project:v0.1.0
+      # image to run
+      image: my-org/my-project:v0.1.0
 
-	  # if set, use component config for the controller-manager
-	  # optional
-	  componentConfig:
-	    # use component config
+      # if set, use component config for the controller-manager
+      # optional
+      componentConfig:
+        # use component config
         enable: true
 
-		# path to component config to put into a ConfigMap
-	    configFilepath: ./path/to/componentconfig.yaml
+        # path to component config to put into a ConfigMap
+        configFilepath: ./path/to/componentconfig.yaml
 
-	  # configure how metrics are exposed
-	  metrics:
-	    # disable the auth proxy required for scraping metrics
+      # configure how metrics are exposed
+      metrics:
+        # disable the auth proxy required for scraping metrics
         # disable: false
 
-		# generate prometheus ServiceMonitor resource
-	    enableServiceMonitor: true
+        # generate prometheus ServiceMonitor resource
+        enableServiceMonitor: true
+
+      serviceAccount:
+        # configure the service account used for RBAC
+        name: foo
+
+        # generate the service account resource
+        generate: true
+
+      serviceAccount:
+	    # configure the service account used for RBAC
+	    name: foo
+
+		# generate the service account resource
```

Comment:
I found at least these 4 tabs
```suggestion
      # configure the service account used for RBAC
      name: foo

    # generate the service account resource
```